### PR TITLE
Fix npm release script

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/matrix-org/matrix-spec/issues/1681

Preview: https://pr1713--matrix-spec-previews.netlify.app

### CODE REVIEW

**Comment by richvdh** [comment]
I have a vague memory of us discussing this before, but I've forgotten everything. For me, and for future archaeologists, could you add some context here: has this always been broken, or did it break recently?

**Comment by richvdh** [comment]
Is this the same as https://github.com/matrix-org/matrix-spec/issues/1681? Or different?

**Comment by t3chguy** [comment]
@richvdh it likely has always been broken, all packages thus far have been manually deployed by me. Good spot on https://github.com/matrix-org/matrix-spec/issues/1681 - updated OP

**Comment by t3chguy** [comment]
Manually published 1.9.0 https://www.npmjs.com/package/@matrix-org/spec

File: changelogs/internal/newsfragments/1713.misc [hunk 1]
```diff
@@ -0,0 +1 @@
+Fix npm release script for `@matrix-org/spec`.
```
Comment (turt2live) [inline changelogs/internal/newsfragments/1713.misc:1]: somehow both CI and I missed this in review: `.misc` doesn't exist in the spec - we use `.clarification`

**Pull request merged**